packs/servicenow/actions/get.py

```diff
@@ -0,0 +1,11 @@
+from lib.actions import BaseAction
+
+
+class GetAction(BaseAction):
+    def run(self, table, query):
+        try:
+            self.client.table = table
+            response = self.client.get(query)
+            return response
+        except e:
```

Comment:
We have no better way right now.

The pattern we use right now is throwing an exception with appropriate error message which causes action to fail.

I know this is not the most ideal approach since you can't that easily differentiate between different exception types (you need to check the message), but more or less the same problem would exist even if we used a different approach.